analysed-packages/atlassian/version-3.41.3/atlassian-3.41.3-SPDX2TV.spdx

```diff
@@ -0,0 +1,371 @@
+﻿SPDXVersion: SPDX-2.3
+DataLicense: CC0-1.0
+
+##-------------------------
+## Document Information
+##-------------------------
+
+DocumentNamespace: http://8a175d67a405/repo/SPDX2TV_site-packages_atlassian-with-distinfo.zip_1722387081.spdx
+DocumentName: /srv/fossology/repository/report
+SPDXID: SPDXRef-DOCUMENT
+
+##-------------------------
+## Creation Information
+##-------------------------
+
+Creator: Tool: fossology-4.4.0.15
+Creator: Person: Kanzlei Jun on behalf of AUDI AG
+CreatorComment: <text>This document was created using license information and a generator from Fossology. It contains the license and copyright analysis of [package]. Please check 'LicenseComments' for explanations of concluded licenses.</text>
+Created: 2024-07-31T00:51:21Z
+LicenseListVersion: 3.22
+
+##-------------------------
+## Package Information
+##-------------------------
+
+
+PackageName: site-packages_atlassian-with-distinfo.zip
+PackageFileName: site-packages_atlassian-with-distinfo.zip
+SPDXID: SPDXRef-upload9056
+PackageDownloadLocation: NOASSERTION
+FilesAnalyzed: true
+PackageVerificationCode: 6f727b7368037c4d5dbbdcbda3b058560582c1f9
+PackageChecksum: SHA1: 2d22a09690b02caf771b729c0b73cb69e0f6b2bb
+PackageChecksum: SHA256: 014f7aa933dbfa7a7d32f1ef5cf51d1470e221aae9530860039436b2729fc7ef
+PackageChecksum: MD5: cef21a9e3482e4b754f22ae220465ac5
+PackageLicenseConcluded: Apache-2.0
+PackageLicenseDeclared: Apache-2.0
+PackageLicenseComments: <text> licenseInfoInFile determined by Scanners:
+ - nomos ("4.4.0.15".5d291e)
+ - monk ("4.4.0.15".5d291e)
+ - ojo ("4.4.0.15".5d291e)
+ - scancode ("4.4.0.15".5d291e)
+ - reso (4.1.0.38.78e82f) </text>
+PackageLicenseInfoFromFiles: NOASSERTION
+PackageCopyrightText: NOASSERTION
+
+Relationship: SPDXRef-DOCUMENT DESCRIBES SPDXRef-upload9056
+
+
+##--------------------------
+## File Information
+##--------------------------
+
+##File
+
+FileName: site-packages_atlassian-with-distinfo.zip/site-packages_atlassian-with-distinfo/atlassian_python_api-3.41.3.dist-info/LICENSE
+SPDXID: SPDXRef-item15754427
+FileChecksum: SHA1: 79dcdbd8306b6c67880c2c586e8a9bbb4ac67ffa
+FileChecksum: SHA256: c5a9809e85f4cdf05a9888a501f3f79d222dedd22f9e3ee8b9e5f5e25eae28f6
+FileChecksum: MD5: e48eab375c6bb2f08058a5f6e33af302
+LicenseConcluded: LicenseRef-fossology-Apache-2.0-af5b3d00d6efa4c915c087dc57f6ae02
+
+LicenseInfoInFile: Apache-2.0
+FileCopyrightText: <text> Copyright 2014 Mateusz Harasymczuk, Gonchik Tsymzhitov </text>
+
+##File
+
+FileName: site-packages_atlassian-with-distinfo.zip/site-packages_atlassian-with-distinfo/atlassian_python_api-3.41.3.dist-info/METADATA

```

Comment:
This file I did not find in the original apckage downloaded from GitHub. In the site-packages folder of my PC there I found it, it contains among other information:
Summary: Python Atlassian REST API Wrapper
Home-page: https://github.com/atlassian-api/atlassian-python-api
Download-URL: https://github.com/atlassian-api/atlassian-python-api
Author: Matt Harasymczuk
Author-email: matt@astrotech.io
Maintainer: Gonchik Tsymzhitov
Maintainer-email: gonchik.tsymzhitov@gmail.com
License: Apache License 2.0
These are the meta data of the package but not the license of this file in my opinion